eogrow/pipelines/training.py

```diff
@@ -59,7 +59,7 @@ class Schema(Pipeline.Schema):
         train_test_split: RandomTrainTestSplitSchema
 
         model_parameters: Dict[str, Any] = Field(
-            default_factory=dict, description="Parameters to be provided to the model"
+            default_factory=Dict[str, Any], description="Parameters to be provided to the model"
```

Comment:
Is such duplication necessary, or does it perhaps automatically infer the default factory from the type, if no default factory is provided?

Reply:
without a default factory the field is required.